tutorials/international/index.md

```diff
@@ -0,0 +1,281 @@
+that the child depends on. The
+[`WidgetsApp`](https://docs.flutter.io/flutter/widgets/WidgetsApp-class.html)
+widget creates a Localizations widget and rebuilds it if the system's locale changes.
+
+You can always lookup an app's current locale with `Localizations.localeOf()`:
+```
+Locale myLocale = Localizations.localeOf(context);
+```
+
+### Loading and Retrieving Localized Values
+
+The `Localizations` widget is used to load and lookup objects that
+contain collections of localized values. Apps refer to these objects
+with [`Localizations.of(context,type)`](https://docs.flutter.io/flutter/widgets/Localizations/of.html).
+If the device's locale changes, the Localizations widget automatically
+loads values for the new locale and then rebuilds widgets that used
+it. This happens because Localizations works like an
+[`InheritedWidget`](https://docs.flutter.io/flutter/widgets/InheritedWidget-class.html). When
+a build function refers to an inherited widget an implicit dependency
+on the inherited widget is created. When an inherited widget changes
+(when the Localizations widget's locale changes), its dependent
+contexts are rebuilt.
+
+Localized values are loaded by the Localizations widget's list of
+[`LocalizationsDelegate`](https://docs.flutter.io/flutter/widgets/LocalizationsDelegate-class.html)s. Each
+delegate must define an async
+[`load()`](https://docs.flutter.io/flutter/widgets/LocalizationsDelegate/load.html)
+method that produces an object which encapsulates a collection of
+localized values. Typically these objects define one method per localized value.
+
+In a large app, different modules or packages might be bundled with
+their own localizations. That's why the Localizations widget manages a
+table of objects, one per LocalizationsDelegate. To retrieve the
+object produced by one of the LocalizationsDelegate's `load` methods,
+you specify a BuildContext and the object's type.
+
+For example the localized strings for the Material widgets are defined
+by the
+[`MaterialLocalizations`](https://docs.flutter.io/flutter/material/MaterialLocalizations-class.html)
+class. Instances of this class are created by a LocalizationDelegate
+provided by the
+[`MaterialApp`](https://docs.flutter.io/flutter/material/MaterialApp-class.html)
+ class. They can be retrieved with `Localizations.of`:
+
+```dart
+Localizations.of<MaterialLocalizations>(context, MaterialLocalizations);
+```
+
+This particular `Localizations.of()` expression is used frequently, so the MaterialLocalizations class provides a convenient shortand:
+
+```dart
+static MaterialLocalizations of(BuildContext context) {
+  return Localizations.of<MaterialLocalizations>(context, MaterialLocalizations);
+}
+
+/// References to the localized values defined by MaterialLocalizations
+/// are typically written like this:
+
+tooltip: MaterialLocalizations.of(context).backButtonTooltip,
+```
+
+### A Class for the App's Localized Resources
```

Comment:
What is the ____-ing equivalent title for this?

Reply:
Maybe we don't need an -ing equivalent.